gradle.properties

```diff
@@ -8,10 +8,10 @@ kotlin.native.ignoreDisabledTargets=true
 org.gradle.jvmargs=-Xmx2G -XX:MaxMetaspaceSize=1G
 
 # SDK
-sdkVersion=0.28.3-SNAPSHOT
+sdkVersion=0.29.0-SNAPSHOT
 
 # codegen
-codegenVersion=0.28.2
+codegenVersion=0.29.0-SNAPSHOT
```

Comment:
Do you mean this should have been 0.28.2-SNAPSHOT before the change?

The tooling will automatically bump the patch version (to 0.28.3, in this case) if there's a SNAPSHOT suffix or not. But since this is a breaking change in codegen, we need to bump the minor version which can't be done automatically.

Reply:
I think he's referring to the `--minor-version-bump` flag to `kat` handling this automatically based on the changelog. I don't actually know and kind of forgot we added that. I'll be reverting the change to `sdkVersion` in `aws-sdk-kotlin` as it will cause conflicts between `main` and `release` if we don't run the merge workflow before next release. 

For this one though will it auto bump both sdk and codegen version? I'm confused why these weren't in sync since I thought thats what we landed on.

Reply:
Yes the `--minor-version-bump` should bump both versions, since the logic for it is located in `bump-version` command which both `sdkVersion` and `codegenVersion` use

Reply:
They weren't in sync because there is a second bump command which runs for `sdkVersion` only. Probably could use a fix